web/client/translations/data.vi-VN.json

```diff
@@ -697,6 +697,7 @@
             "textFormatTitle": "VĂN BẢN",
             "title": "Tiêu đề",
             "titleTranslations": "Bản dịch tiêu đề",
+            "heightOffset": "Height offset (m)",
```

Comment:
no need to add locales messages, even if not translated, to not supported locales :)